forc-plugins/forc-client/Cargo.toml

```diff
@@ -19,11 +19,13 @@ forc-pkg = { version = "0.40.1", path = "../../forc-pkg" }
 forc-tracing = { version = "0.40.1", path = "../../forc-tracing" }
 forc-tx = { version = "0.40.1", path = "../forc-tx" }
 forc-util = { version = "0.40.1", path = "../../forc-util" }
+fuel-abi-types = "0.3"
 fuel-core-client = { workspace = true, default-features = false }
 fuel-crypto = { workspace = true }
 fuel-tx = { workspace = true, features = ["builder"] }
 fuel-vm = { workspace = true }
 fuels-accounts = { workspace = true }
+fuels-code-gen = "0.42" 
```

Comment:
I will be removing this dependency as we don't need it anymore with latest release of `fuel-abi-types`